tpm/commands.py

```diff
@@ -4,6 +4,7 @@
 import time
 
 from kpi.command import *
+from online_player_api import *
```

Comment:
`import *` 是我之前写代码留下的坏习惯，还是尽量不用的好

```suggestion
from online_player_api import check_online
```

Reply:
明白！